### DISCUSSION TITLE
Android QField availability on F-Droid

### DISCUSSION THREAD

**Discussion by gdt** [opengisch/QField#2706] [Ideas]
The preferred app store for Free Software on Android is https://f-droid.org/.   It also functions as a directory of Free Software android apps.  F-Droid has a number of nice properties:
  * no tracking, no login required, no restriction of apps by geo region, no censorship by google
  * peer to peer app sharing without internet
  * all content is built by f-droid to ensure that it corresponds to source and is entirely Free Software
  * f-droid declares antifeatures if there are issues in the code (trackers, promotion of non-free network services), allowing people to know what they are getting into
  * the usual checking for and installing updates

Many open geo apps are there, but QField isn't.   Assuming there's no google play services, it should be possible.   While I see that there are release apks, those don't update, and they are missing the third-party review/labeling that gives confidence that the strict Free Software terms are met.  (Sometimes apps end up with flavors without Google Play services, because while they proclaim that the app is Free Software the binary apk isn't even if the sources are.)

So it would be great to have QFIeld in f-droid for the following reason:
  * discoverability by F-Droid users, listing in the premier directory of Free Software Android aps
  * updating for those that don't use Google Play
  * evidence of QField being 100% Free Software

Examples:
  * https://f-droid.org/en/packages/de.blau.android/
  * https://f-droid.org/en/packages/net.osmand.plus/
  * https://f-droid.org/en/packages/com.android.gpstest.osmdroid/
  * https://f-droid.org/en/packages/app.organicmaps/
  * https://f-droid.org/en/packages/de.dennisguse.opentracks/
  * https://f-droid.org/en/packages/com.kylecorry.trail_sense/

**Comment by mbernasocchi** [comment]
Hi @gdt,
thanks for your analisis. it would indeed be great if QField was in f-droid but honestly, for now we prefer invest our limited resources on making QField better and better. 
Obviously, if someone (you?) would be willing to contribute the effort we'd gladly help with suggestions and the likes.

cheers
Marco

**Comment by m-kuhn** [comment]
There is a [rfp open on F-Droid](https://gitlab.com/fdroid/rfp/-/issues/249) please contribute there. 

If there is any information required to make this happen, the QField dev team is more than happy to help out!

Meanwhile, the apks are available for download without google play from the [release page](https://github.vom/opengisch/QField/releases)

**Comment by ZsanettMed** [comment]
Hi @gdt!

Hello,

I've posted this on our [QField Idea Platform](https://ideas.qfield.org/app-feature-requests/p/qfield-on-f-droid), where you and others can support it by voting. Since we're gathering all ideas on the platform, I will be closing this thread.

Best ,
Zsanett
